src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -90,6 +91,12 @@ def load_arguments(self, _):
                    help="Create your Azure Spring Cloud service in an Azure availability zone or not, "
                         "this could only be supported in several regions at the moment.",
                    default=False, is_preview=True)
+        c.argument('build_pool_size',
+                   arg_type=get_enum_type(['S1', 'S2', 'S3', 'S4', 'S5']),
+                   validator=validate_build_pool_size,
+                   default='S1',
+                   is_preview=True,
+                   help='(Enterprise Tier Only) Size of build agent pool. See Azure Spring Cloud Doc for size info.')
```

Comment:
May I ask you to add the Doc url in help info ?

Reply:
The doc has not been published yet.

Reply:
Updated.